### PR TITLE
Rework Docker: Python 3 + Remove GDAL

### PR DESCRIPTION
**Pull request by DavidLemayian**
## Description

This PR reworks Docker and how we use Kenya as default app for easier development of the package and possibly smaller docker images.

This also switches to Python 3.

And removes GDAL because we've implemented mapit.

Lastly, we get rid of the `hurumap-dashboard` image.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Screenshots

N/A

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by DavidLemayian** [comment]
@phillipahereza Made it Python3 compatible on this branch - https://github.com/CodeForAfricaLabs/wazimap/tree/feature/python3

File: contrib/docker/release-build.sh [hunk 1]
```diff
@@ -5,4 +5,4 @@ USERNAME=codeforafrica
 # image name
 IMAGE=hurumap
 docker build --target hurumap --no-cache -t $USERNAME/$IMAGE:latest .
-docker build --no-cache -t $USERNAME/$IMAGE-dashboard:latest .
+docker build --target hurumap-dashboard --no-cache -t $USERNAME/$IMAGE-dashboard:latest .
```
Comment (kilemensi) [inline contrib/docker/release-build.sh]: May be we can reuse `$IMAGE`?:
```shell
 docker build --target $IMAGE --no-cache -t $USERNAME/$IMAGE:latest .
 docker build --target $IMAGE-dashboard --no-cache -t $USERNAME/$IMAGE-dashboard:latest .
```

File: hurumap/dashboard/urls.py [hunk 2]
```diff
@@ -20,4 +21,4 @@
                   url(r'^dashboard/', include(wagtailadmin_urls)),
                   url(r'^documents/', include(wagtaildocs_urls)),
                   url(r'^', include(wagtail_urls)),
-              ] + hurumap_urlpatterns
+              ]
```
[the remark this answers is not in the document]
Comment (DavidLemayian) [reply hurumap/dashboard/urls.py]: Yup. Just realised this.

File: contrib/docker/release-build.sh (hunk 1 above)
Comment (DavidLemayian) [reply contrib/docker/release-build.sh]: Cool will do.

**Comment by DavidLemayian** [comment]
@kilemensi we won't be keeping 2.7 compatibility. I'm not sure why those show up but could you clean them out for me and merge when ready?

**Pull request merged**